src/elementary/test/elementary.test.cpp

```diff
@@ -59,3 +61,70 @@ SCENARIO( "emit" ) {
     } // THEN
   } // GIVEN
 } // SCENARIO
+
+SCENARIO( "join" ) {
+
+  GIVEN( "a vector of strings" ) {
+
+    std::vector< std::string > empty = {};
+    std::vector< std::string > one = { "ab" };
+    std::vector< std::string > strings = { "ab", "cd", "ef" };

```

Comment:
It makes me sad you couldn’t come up with something more fun than this. There are so many things you could have done instead, like favorite muppet characters, people who annoy you, worst European cities, etc.

Reply:
At least I know my ABCs ;-)